README.md

```diff
@@ -2,51 +2,60 @@
 
 # FINOS | TraderX Example of a Simple Trading App
 
-![DEV Only Warning](https://badgen.net/badge/warning/not-for-production/red) ![Local Dev Machine Supported](http://badgen.net/badge/local-dev/supported/green)
+![DEV Only Warning](https://badgen.net/badge/warning/not-for-production/red) ![Local Dev Machine Supported](https://badgen.net/badge/local-dev/supported/green)
 
-The Sample Trading Application is a usable simple yet distributed reference application
-in the financial services domain which can be used for experimenting with various
-techniques and other open source projects.  It is designed to be simple and accessible
-to programmers of all backgrounds, and can serve as a starting point for educational
-and experimentation purposes.
+<img align="right" src="./docs/img/2023_TraderX_Vertical.png" alt="TraderX Logo" width="250"/>
+
+TraderX is a Sample Trading Application, designed to be a distributed reference application 
+in the financial services domain which can serve as a starting point for experimentation 
+with various techniques and other open source projects.  It is designed to be simple
+and accessible to developers of all backgrounds, with minimal pre-assumptions, and it 
+can serve as a starting point for educational and experimentation purposes.
 
 It is designed to be runnable from any developer workstation with minimal assumptions
-other than Node, Java and Python runtimes.
+other than Node, Java and Python runtimes. The libraries and toolkits it uses are meant
+to be as vanilla as possible, to preserve its approachability by developers of all levels.
 
 It contains Java, NodeJS, Python, .NET components that communicate over REST APIs and
 messaging systems and are able to showcase a wide range of technical challenges to solve.
 
 More detailed information about this project can be found in the website which is generated
 from the code under the `docs` directory of this project.
 
-## Current Project Status
 
-This is currently a Work-In-Progress. At the moment there are some components which are just placeholder API contracts for an implementation to be created, while others are already runnable reference implementations.
+## Project Demo and Overview Presentation
+
+Learn more about the project - including a brief demo, in the Keynote Demo session 
+that was presented at the [Open Source in Finance Forum 2023](https://events.linuxfoundation.org/open-source-finance-forum-new-york/)
+
+[![FINOS - Incubating](./docs/img/2023_osff_video_thumb.png)](https://youtu.be/fO-T3V-GKBs?t=326)
```

Comment:
Small change but I would change this to say `TraderX - Demo` instead of `FINOS - Incubating`

Reply:
Thanks. Fixed